webapp/src/main/java/com/box/l10n/mojito/service/pollableTask/PollableTaskBlobStorage.java

```diff
@@ -46,8 +50,13 @@ public String getOutputJson(Long pollableTaskId) {
     String outputJson =
         structuredBlobStorage
             .getString(POLLABLE_TASK, outputName)
-            .orElseThrow(
-                () -> new RuntimeException("Can't get the output json for: " + pollableTaskId));
+            .orElseGet(
+                () -> {
+                  logger.warn(
+                      "Can't get the output json for pollable id {}, assuming task has been skipped, returning empty object.",
+                      pollableTaskId);
+                  return "{}";
+                });
```

Comment:
@aurambaj Issue I had with this approach is that if you mark a pollable task as finished and the caller expects a returned output object then we were throwing the above Exception as no output exists as the pollable task has been skipped. We don't have this issue with #84 as by returning a previous pollable we can guarantee a return.

In the above new code I just return an empty object, any thoughts on this? I don't see an easy way to avoid this, I can update the return type to an `Optional` to make this clearer to callers but it will also require a large amount of code updates to update the existing callers to handle the possibility of empty output.